landing-page/content/common/index-and-statistics-format.md

```diff
@@ -0,0 +1,129 @@
+are documented at [Blob types](#blob-types).
+
+## Format specification
+
+A file conforming to the format specification should have the structure as
+described below.
+
+### File structure
+
+The file has the following structure
+
+```
+Magic Blob₁ Blob₂ ... Blobₙ Footer
+```
+
+where
+
+- `Magic` is four bytes 0x50, 0x46, 0x49, 0x53 (short for: Plain Format for
+  Indices and Statistics),
+- `Blobᵢ` is i-th blob contained in the file, to be interpreted by application
+  according to the footer,
+- `Footer` is defined below.
+
+### Footer structure
+
+Footer has the following structure
+
+```
+Magic FooterPayload FooterPayloadSize Reserved Flags FileFormatVersion Magic
+```
+
+where
+
+- `Magic` four bytes, same as at the beginning of the file.
+- `FooterPayload` optionally compressed, UTF-8 encoded JSON payload describing the
+  blobs in the file, with the structure described below,
+- `FooterPayloadSize` is a length in bytes of the `FooterPayload` (compressed),
+  stored as 4 byte integer,
+- `Reserved` is 4 bytes reserved for future use, currently should be written as
+  0x00, 0x00, 0x00, 0x00,
+- `Flags` a 4 byte integer for boolean flags
+  - 0 (lowest bit): whether `FooterPayload` is compressed
+  - all other bits are reserved for future use and should be set to 0 on write
+- `FileFormatVersion` is a number, stored as 4 byte integer.
+
+A 4 byte integer is always signed, in a two's complement representation, stored
+little-endian.
+
+### Footer Payload
+
+Footer payload bytes is either uncompressed or LZ4-compressed (as a single LZ4
+compression frame with content size present), UTF-8 encoded JSON payload
+representing a single `FileMetadata` object.
+
+#### FileMetadata
+
+`FileMetadata` has the following fields
+
+
+| Field Name | Field Type                              | Required | Description |
+| ---------- | --------------------------------------- | -------- | ----------- |
+| blobs      | list of BlobMetadata objects            | yes      |
+| properties | JSON object with string property values | no       | storage for arbitrary meta-information, like writer identification/version
+
+#### BlobMetadata
+
+`BlobMetadata` has the following fields
+
+| Field Name | Field Type             | Required | Description |
+| ---------- | ---------------------- | -------- | ----------- |
+| type       | JSON string            | yes      | See [Blob types](#blob-types)
+| columns    | list of JSON long      | yes      | list of column IDs the blob was computed for
+| offset     | JSON long              | yes      | The offset in the file where the blob contents start. Reader should assume the value can be more than 2^32.
+| length     | JSON long              | yes      | The length of the blob stored in the file
+| compression_codec | JSON string     | no       | See [Compression codecs](#compression-codecs). If omitted, the data is assumed to be uncompressed.
+
+### Blob types
```

Comment:
I think some of the requirements in the `apache-datasketches-theta-v1` blob type should be moved to a new section on computing blobs. Specifically, we should standardize how values are computed to feed into a sketch algorithm.

Reply:
We have two sketches defined at the moment. For one of them it matters, for the other it does not. 
I'd suggest we reword and extract common parts once we have more cases, so that we capture the commonalities well.